Makefile

```diff
@@ -0,0 +1,23 @@
+CXXFLAGS = -W -Wall -Wextra -std=c++11
+LDFLAGS = -l boost_system -l pthread
```

Comment:
Please don't put a whitespace after `-l` flag. Should be: `-lboost_system -lpthread`.